From akpm@osdl.org Sat Jul 16 21:13:51 2005
Date: Sat, 16 Jul 2005 21:07:02 -0700
From: Andrew Morton <akpm@osdl.org>
Cc: Jesper Juhl <jesper.juhl@gmail.com>
To: Greg KH <greg@kroah.com>
Subject: SPEAKUP: make SPEAKUP_DEFAULT depend on SPEAKUP
Message-Id: <20050716210702.496b401d.akpm@osdl.org>


When running `make oldconfig' with my 2.6.13-rc3 config on
2.6.13-rc3-mm1 I got a bit surprised when I answered `N'o to
CONFIG_SPEAKUP but then still got prompted for CONFIG_SPEAKUP_DEFAULT -
that didn't make sense, why would I want to select a default synthesizer
for speakup if I have disabled speakup alltogether in the first place?
The patch below makes SPEAKUP_DEFAULT depend on SPEAKUP. I believe
that's the sane thing to do :)


Signed-off-by: Jesper Juhl <jesper.juhl@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/speakup/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/char/speakup/Kconfig
+++ gregkh-2.6/drivers/char/speakup/Kconfig
@@ -205,6 +205,7 @@ if SPEAKUP != n
 endif
 
 config SPEAKUP_DEFAULT
+	depends on SPEAKUP
 	string "Choose Default synthesizer for Speakup"
 	default "none"
 
